pcdsdevices/dccm.py

```diff
@@ -0,0 +1,208 @@
+        In addition, there is serious potential to fry the motor itself,
+        as it is run too intensively.
+        """
+        if kill:
+            # Must always wait if we are killing the PID
+            wait = True
+        st = super().move(*args, wait=wait, **kwargs)
+        time.sleep(0.01)  # safety wait. Necessary?
+        if kill:
+            self.theta.kill()
+        return st
+
+
+
+class DCCM(BaseInterface, GroupDevice):
+    """
+    The full DCCM assembly.
+
+    Double Channel Cut Monochrometer controlled with a Beckhoff PLC.                                                                                             
+    This includes five axes in total:                                                                                                                            
+        - 2 for crystal manipulation (TH1/Upstream and TH2/Downstream)                                                                                           
+        - 1 for chamber translation in x direction (TX)                                                                                                          
+    - 2 for YAG diagnostics (TXD and TYD)                                                                                                                        
+                                                                                                                                                                 
+    Parameters                                                                                                                                                   
+                                                                                                                                                                 
+    ----------                                                                                                                                                   
+                                                                                                                                                                 
+    prefix : str,optional                                                                                                                                        
+    Base PV for DCCM motors                                                                                                                                      
+    name : str                                                                                                                                              
+    
+    This requires a huge number of motor pv prefixes to be passed in.
+    Pay attention to this docstring because most of the arguments are in
+    the kwargs.
+
+    Parameters
+    ----------
+    prefix : str, optional
+        Devices are required to have a positional argument here,
+        but this is not used. 
+
+    
+    """
```

Comment:
This parameters section is malformed, incorrect, and duplicated, possibly from a copy/paste saga
```suggestion
    Parameters
    ----------
    prefix : str
        Base PV for DCCM motors
    name : str, keyword-only
        name to use in bluesky
    """
```